.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
hmm: I normally list the complete ".idea" folder in my root .gitignore

A nice test of your git skills: remove the .idea folder fromthis branch and add the .idea folder to the root ignore file.

Reply:
ok i will do this